cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -94,6 +94,8 @@
       cronjob at:'* * * * *', do:deploy_dir('bin', 'cron', 'confirm_usage')
       cronjob at:'0 */2 * * *', do:deploy_dir('bin', 'cron', 'teacher_applications_to_gdrive')
       cronjob at:'30 */2 * * *', do:deploy_dir('bin', 'cron', 'summer_workshops_to_gdrive')
+      # [Clare] Disabled so that we can do the first run manually on production-daemon, to update credentials.
```

Comment:
can you elaborate on this? is the plan to re-enable it in a followup PR?

Reply:
Yes! I will run it manually the first time, and if all goes well, I'll submit a follow up to enable this.